test/functional/plugin/man_spec.lua

```diff
@@ -11,7 +19,8 @@ describe(':Man', function()
       clear()
       command('syntax on')
       command('set filetype=man')
-      command('syntax off')  -- Ignore syntax groups
+      command('syntax off')  -- Ignore syntax groups.
+      command('unlet b:man_pager') -- Allow  man#init_pager to rerun.
```

Comment:
i would guess this `unlet` is causing the crash, did you try without it?

Reply:
I'll give it a shot but why would this be causing the crash, and why only on windows/big endian.